tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -601,7 +499,6 @@ def get_nodes_status(self):
 def get_scheduler_commands(scheduler, remote_command_executor):
     scheduler_commands = {
         "awsbatch": AWSBatchCommands,
-        "sge": SgeCommands,
         "slurm": SlurmCommands,
         "torque": TorqueCommands,
```

Comment:
don't you need to remove Torque as well?

Reply:
Nope, see PR description:
> TorqueCommands in tests/common/schedulers_common has been preserved because in test_slurm we're testing torque wrappers for slurm.